.gitpod.yml

```diff
@@ -9,11 +9,13 @@ tasks:
       eval $(gp env -e API_URL=$(gp url 3000))
       eval $(gp env -e WEB_URL=$(gp url 8000))
       eval $(gp env -e API_HOST=$(gp url 8443))
+      eval $(gp env -e SSO_URL=$(gp url 8081))
```

Comment:
I don't think this file should be changing as part of this PR, should it?

Reply:
I wanted to make the singup flow works for people who might review this PR using Gitpod.